Source: John Kallimanis
Upstream: Not applied by upstream. Created for porting reasons.
Reason: Link objects to ncurses instead of termcap. Respect env vars.
*** ephem/Makefile	2014-04-09 18:31:04.153318724 +0300
--- ephem-4.30/Makefile	2014-04-09 18:14:46.744538015 +0300
***************
*** 1,7 ****
  # Makefile for ephem, v 4.30
  
- CFLAGS = -O
- 
  EPHEM=	aa_hadec.o altj.o altmenus.o anomaly.o cal_mjd.o circum.o comet.o \
  	compiler.o constel.o eq_ecl.o flog.o formats.o io.o listing.o main.o \
  	mainmenu.o moon.o moonnf.o nutation.o objx.o obliq.o parallax.o \
--- 1,5 ----
***************
*** 10,16 ****
  	version.o watch.o
  
  ephem:	$(EPHEM)
! 	cc -o $@ $(EPHEM) -ltermcap -lm
  
  aa_hadec.o:	astro.h
  
--- 8,14 ----
  	version.o watch.o
  
  ephem:	$(EPHEM)
! 	$(CC) $(CFLAGS) -o $@ $(EPHEM) -lcurses -lm $(LDFLAGS) 
  
  aa_hadec.o:	astro.h
  
